src/notation/qml/MuseScore/NotationScene/internal/PercussionPanelPad.qml

```diff
@@ -23,87 +23,108 @@ import QtQuick 2.15
 
 import Muse.Ui 1.0
 import Muse.UiComponents 1.0
+import Muse.GraphicalEffects 1.0
 import MuseScore.NotationScene 1.0
 
 Rectangle {
     id: root
 
     property var padModel: null
 
+    property bool isEmptySlot: Boolean(root.padModel) ? root.padModel.isEmptySlot : true
```

Comment:
```suggestion
    readonly property bool isEmptySlot: Boolean(root.padModel) ? root.padModel.isEmptySlot : true
```

Reply:
As far as I can tell, it's a private property for this component. So it's better to declare it inside QObject { id: prv }

Reply:
(will resolve in next Percussion Panel PR)